jps-plugin/src/org/jetbrains/jps/incremental/scala/data/CompilationData.scala

```diff
@@ -213,4 +219,19 @@ object CompilationData {
             "\nPlease configure separate output paths to proceed with the compilation." +
             "\nTIP: you can use Project Artifacts to combine compiled classes if needed.")
   }
+
+  private def scalaVersionFrom(compilerClasspath: Seq[File]): String = {
```

Comment:
I wonder if there's a simple way to get the Scala version. Perhaps the `ModuleChunk` has something?

Reply:
I don't know if that is possible. This was suggested on the slack channel, but I will look if there is some other way.

Reply:
Found a much nicer way to find the Scala version. :)